bin/setup.bat

```diff
@@ -32,6 +32,7 @@ if defined LS_JAVA_HOME (
   if exist "%LS_HOME%\jdk" (
     echo WARNING: Using JAVA_HOME while Logstash distribution comes with a bundled JDK.
```

Comment:
Shouldn't this also be updated? and/or also take LS_JAVA_HOME into consideration?

Reply:
The deprecation notice is written at subsequent message, so I wouldn't write it twice. This warning is specific to for the usage of env variable when the LS package comes with a bundled JDK.

With this message the user sees on his console:
```
WARNING: Using JAVA_HOME while Logstash distribution comes with a bundled JDK.
DEPRECATION: The use of JAVA_HOME is now deprecated and will be removed starting from 8.0. Please configure LS_JAVA_HOME instead.
```
So he's fully informed of the deprecation and the usage of a system JDK while Logstash is bundled with a JDK.